src/main/java/com/google/api/codegen/transformer/php/PhpGapicSurfaceTransformer.java

```diff
@@ -453,7 +465,9 @@ private void addApiImports(GapicInterfaceContext context) {
     typeTable.saveNicknameFor("\\Google\\ApiCore\\CredentialsWrapper");
     typeTable.saveNicknameFor("\\Google\\ApiCore\\GapicClientTrait");
     typeTable.saveNicknameFor("\\Google\\ApiCore\\PathTemplate");
-    typeTable.saveNicknameFor("\\Google\\ApiCore\\RequestParamsHeaderDescriptor");
+    if (supportsGrpcTransport()) {
```

Comment:
tiny nit: Does order matter here? If not, I would move this to be the least one, for readability.

Reply:
I agree! Order doesn't matter! Moved for readability.